fjord/feedback/tests/test_models.py

```diff
@@ -322,6 +324,8 @@ def test_purge(self):
         eq_(ResponseContext.objects.filter(
             opinion__created__gte=cutoff).count(),
             5)
+        eq_(pre_journal_count+1,
+            Record.objects.filter(action='purge_data').count())
```

Comment:
Since this is just a test, we can get rid of the query that finds the count by using our knowledge that there will be just one journal entry created when the testcase is run. So it can be rewritten as

    eq_(1,
            Record.objects.filter(action='purge_data').count())

I will also test for the value of the msg.